src/common/SConscript

```diff
@@ -24,6 +24,11 @@ def build_daos_common(env, client, prereqs):
         denv.Append(OBJPREFIX="v_")
         libname = 'libdaos_common_pmem'
         prereqs.require(denv, 'pmdk')
+        if denv["STACK_MMAP"] == 1:
+            common_libs.extend(['abt'])
+            COMMON_FILES.extend(['stack_mmap.c'])
```

Comment:
I guess this works because the pmem version is built 2nd but maybe a comment where we call this function to makes sure it stays that way would be good.

Reply:
ok, I got it and thx, will change this (duplicate COMMON_FILES in a local version I ca extend for poem build only...) in a next commit if any further review requires a more critical change....